packages/cli/src/InternalHooks.ts

```diff
@@ -262,6 +264,9 @@ export class InternalHooks implements IInternalHooksClass {
 					isManual: executionMode === 'manual',
 					workflowId: workflowData?.id?.toString(),
 					workflowName: workflowData?.name,
+					metaData: executionMetadata?.map((metadata) => ({
```

Comment:
This is gonna look like an array of key value pairs - is this what we expect?
We might want to use reduce instead if we want to have a single key-value object.